notebooks/supernova_model_sims.ipynb

```diff
@@ -2,79 +2,380 @@
  "cells": [
```

Comment:
<p>I am surprised the magnitudes are flat at a specific value at the beginning (I'd expected it to have zero flux -&gt; infinite magnitude)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>